### ISSUE TITLE
trying to understand pricing

### ISSUE THREAD

**Issue by rounders** [librato/librato-rails#119]
I recently installed the librato-rails gem and am successfully receiving stats about my rails app in my Librato account. My app runs on 3 servers so I would expect, based on the gem's README, that this would generate 3 streams. However, after collecting stats for a day or so I saw my estimated monthly charge go up by quite a bit more than expected. Upon digging further I saw Librato was detecting 18.6 sources for 'rails.view.render_partial.time' and 15 or more sources for some of the other metrics too. 
I emailed Librato support to inquire and was told that each partial is a source and that I would be charged for each source count for that metric. 

I think we need some clarity around this issue. A typical rails app can have a lot of partials and so if each one results in a source that gets charged separately then this could result in a large increase in ones monthly bill.

**Comment by nextmat** [comment]
Agreed. Especially with certain app designs this can cause an explosion in sources. 

As of v1.3.0 you can [enable/disable specific instrumentation sets](https://github.com/librato/librato-rails#automatic-measurements). So if you haven't already you might want to drop `-rails_render` into your config.

I've been thinking we should probably roll a new major ver with render stats disabled by default. Anyone else have thoughts on this issue? This instrumentation is useful for a lot of users so I don't think it should be disabled without a version bump & some README context if we decide to do that.

**Comment by rounders** [comment]
I did remove rails_render from my config and that worked great. For me the only issue here is ensuring that the issue with rails_render is made clear on the README.md.  The 'Cross-Process Aggregation' section gives the impression that one will get charged for 1 stream per host. So some mention that this is not the case with rails_render would fix that.
